src/databricks/labs/ucx/hive_metastore/workflows.py

```diff
@@ -26,14 +38,9 @@ def migrate_dbfs_root_delta_tables(self, ctx: RuntimeContext):
         """
         ctx.tables_migrator.migrate_tables(what=What.DBFS_ROOT_DELTA, acl_strategy=[AclMigrationWhat.LEGACY_TACL])
 
-
-class MigrateTablesInMounts(Workflow):
-    def __init__(self):
-        super().__init__('migrate-tables-in-mounts-experimental')
-
-    @job_task
-    def scan_tables_in_mounts_experimental(self, ctx: RuntimeContext):
-        """[EXPERIMENTAL] This workflow scans for Delta tables inside all mount points
-        captured during the assessment. It will store the results under the `tables` table
-        located under the assessment."""
-        ctx.tables_in_mounts.snapshot()
+    @job_task(job_cluster="table_migration", depends_on=[ScanTablesInMounts.scan_tables_in_mounts_experimental])
+    def migrate_tables_in_mounts_experimental(self, ctx: RuntimeContext):
```

Comment:
can we have it in a separate experimental workflow? if it's in the `migrate-tables` workflow, then we have no guarantee that `ScanTablesInMounts.scan_tables_in_mounts_experimental` was executed before